opnsynid_receivable_aging_aeroo_report/report/query_receivable_aging.py

```diff
@@ -0,0 +1,228 @@
+# -*- coding: utf-8 -*-
+# © 2015 OpenSynergy Indonesia
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, api
+from openerp.tools import drop_view_if_exists
+from datetime import datetime
+
+
+class QueryReceivableAging(models.Model):
+
+    _name = 'account.query_receivable_aging'
+    _description = 'Query Receivable Aging'
+    _auto = False
+
+    @api.one
+    def function_aging(self):
```

Comment:
@api.multi
def function_aging(self):
    for line in self:
        ...
        ....
        # cara returnnya
        line.aging1 = 0.0